js/templates/modals/about/btcticker.html

```diff
@@ -0,0 +1,5 @@
+<div class="pureFlex cent currentBTCPrice clrP clrBr clrSh3 tx4 contentBox padSm">
```

Comment:
It doesn't look like the `currentBTCPrice` is being used anywhere. If that's correct, can we remove it then please?